src/common/logger.py

```diff
@@ -13,8 +13,8 @@
     'formatters': {
         'colored': {
             '()': 'colorlog.ColoredFormatter',
-            'fmt': '%(log_color)s%(asctime)s %(levelname)-10s%(message)s%(reset)s',
-            'datefmt': '%y/%m/%d %H:%M:%S',
+            'fmt': '%(log_color)s%(levelname)-10s%(message)s%(reset)s',
```

Comment:
```suggestion
            'fmt': '%(log_color)s%(message)s%(reset)s',
```
The color is now enough to distinguish levels:
![image](https://github.com/user-attachments/assets/e5621e2f-da74-45d5-af0e-af4d16529ebc)

Reply:
Not convinced the colours can be distinguished properly when you are colourblind.
I won't remove the level.
At most, I will put it at the start of the line.

Reply:
![téléchargement](https://github.com/user-attachments/assets/3b644bdd-1154-40b7-ac1f-b992e99cb0a9)
With deuteranopia (green-blind colourblindness, simulated above), you can barely differentiate warning and error